helm/docs/common-configuration.md

```diff
@@ -0,0 +1,450 @@
+      </td>
+      <td>
+       The type of kubernetes cluster provider you are using. In this case the clusterProvider value is <i>onpremise</i>
+      </td>
+    </tr>
+    <tr>
+      <td>
+       persistentVolume.onpremise.hostPath
+      </td>
+      <td>
+       The shared folder path in your host machine.<br/><br/>
+       Ex: Create a folder in your machine to store the shared folders for application usage.
+       <i>D://app/shared</i><br/><br/>
+       The corresponding value for the above shared folder will be,
+       <i>/run/desktop/mnt/host/d/app/shared</i>
+      </td>
+    </tr>
+</table>
+<br/>
+
+## Image
+
+We need to give a image tag for Bold BI, Bold Reports and ID. By default set a latest version of a image tags. You can change a tag based on requirments and the tag overides the image tag whose default in the chart appVersion.<br>
+```console
+image:
+  idRepo: gcr.io/boldbi-294612
+  biRepo: gcr.io/boldbi-294612
+  reportsRepo: gcr.io/boldreports
+  # Overrides the image tag whose default is the chart appVersion.
+  idTag: 5.2.48
+  biTag: 5.2.48
+  reportsTag: 4.2.52
+````
+Repository details are available in image section to refer the image tags. No need to change the repository details for idRepo, biRepo and reportsRepo.
+
+## Version
+
+By default latest version of Bold BI, Bold Reports and IDP in the version section. You can change a version based on requirements.
+```console
+versions:
+  idp: "3.1.21"
+  bi: "5.2.48"
+  reports: "4.2.52"
+````
+> **NOTE:**  We need give a latest version of `idp` when you have latest version of Bold BI or Bold Reports version
+
+## Load Balancing
+
+### Ingress-Nginx
+
+By default Nginx is used as reverse proxy for Bold BI. To configure load balancing in Bold BI take a look at the below snipet. 
+
+```console
+loadBalancer:
+  type: nginx
+```
+
+The default `ingressClassName` is `nginx`. Please refer [here](https://kubernetes.io/docs/concepts/services-networking/ingress/#deprecated-annotation) for more details.
+
+```console
+loadBalancer:
+  # For Kubernetes >= 1.18 you should specify the ingress-controller via the field ingressClassName
+  # See https://kubernetes.io/blog/2020/04/02/improvements-to-the-ingress-api-in-kubernetes-1.18/#specifying-the-class-of-an-ingress
+  # ingressClassName: nginx
+```
+
+### Istio Ingress Gateway
+
+If you need to configure Bold BI with Istio then you can change the value as `istio` in your configuration like below.
+
+```console
+loadBalancer:
+  type: istio
+```
+
+### sticky session
+
+You can change the affinityCookieExpiration time. The default value is 600s.
+
+```console
+loadBalancer:
+  affinityCookie:
+    enable: true
+    affinityCookieExpiration: 600
+```
+
+### SSL Configuration
+
+If you have the SSL certificate for your DNS and need to configure the site with your SSL certificate, just pass your DNS with `https` protocol to `appBaseUrl`, by doing this it will automatically enable SSL in both Ingress and Istio.
```

Comment:
If you have the SSL certificate for your DNS and need to configure the site with your SSL certificate, just pass your DNS with the `https` protocol to `appBaseUrl`, by doing this it will automatically enable SSL in both Ingress and Istio.

Reply:
concern addressed